src/main/flight/failsafe.h

```diff
@@ -34,7 +36,9 @@ typedef struct failsafeConfig_s {
     uint8_t failsafe_kill_switch;           // failsafe switch action is 0: identical to rc link loss, 1: disarms instantly
     uint16_t failsafe_throttle_low_delay;   // Time throttle stick must have been below 'min_check' to "JustDisarm" instead of "full failsafe procedure".
     uint8_t failsafe_procedure;             // selected full failsafe procedure is 0: auto-landing, 1: Drop it
-} failsafeConfig_t;
+} PG_PACKED failsafeConfig_t;
```

Comment:
Thanks for looking at this. I've removed the PG_PACKED attribute and re-ordered the structure members (I know the compiler does this, but I used to manually align in the old days...). I only put PG_PACKED in because I was copying from Cleanflight. I think I'll avoid it altogether, since it's benefits are fairly marginal.